packages/arb-ts/integration_test/testHelpers.ts

```diff
@@ -60,3 +83,77 @@ export const warn = (text: string) => {
   console.log(chalk.red(`WARNING: ${text}`))
   console.log()
 }
+
+export const instantiateRandomBridge = () => {
```

Comment:
should we rename this to make it clear that the bridge itself is not random but rather the PK used in it